testapi.pm

```diff
@@ -375,7 +384,7 @@ Example:
 Returns true if screen changed or C<undef> on timeout. Default timeout is 10s.
 
 =cut
-sub wait_screen_change(&@) {
+sub wait_screen_change {
```

Comment:
removed this, the "no critic" line did not work because I erroneously put it into the POD, fixed.